sssom/cliquesummary.py

```diff
@@ -47,111 +69,125 @@ class Count(int):
 
 
 class String(str):
-    """ A character string """
+    """A character string"""
+
     type_class_uri = XSD.string
     type_class_curie = "xsd:string"
     type_name = "string"
     type_model_uri = SSSOM.CS.String
 
 
 class Integer(int):
-    """ An integer """
+    """An integer"""
+
     type_class_uri = XSD.integer
     type_class_curie = "xsd:integer"
     type_name = "integer"
     type_model_uri = SSSOM.CS.Integer
 
 
 class Boolean(Bool):
-    """ A binary (true or false) value """
+    """A binary (true or false) value"""
+
     type_class_uri = XSD.boolean
     type_class_curie = "xsd:boolean"
     type_name = "boolean"
     type_model_uri = SSSOM.CS.Boolean
 
 
 class Float(float):
-    """ A real number that conforms to the xsd:float specification """
+    """A real number that conforms to the xsd:float specification"""
+
     type_class_uri = XSD.float
     type_class_curie = "xsd:float"
     type_name = "float"
     type_model_uri = SSSOM.CS.Float
 
 
 class Double(float):
-    """ A real number that conforms to the xsd:double specification """
+    """A real number that conforms to the xsd:double specification"""
+
     type_class_uri = XSD.double
     type_class_curie = "xsd:double"
     type_name = "double"
     type_model_uri = SSSOM.CS.Double
 
 
 class Decimal(Decimal):
-    """ A real number with arbitrary precision that conforms to the xsd:decimal specification """
+    """A real number with arbitrary precision that conforms to the xsd:decimal specification"""
+
     type_class_uri = XSD.decimal
     type_class_curie = "xsd:decimal"
     type_name = "decimal"
     type_model_uri = SSSOM.CS.Decimal
 
 
 class Time(XSDTime):
-    """ A time object represents a (local) time of day, independent of any particular day """
+    """A time object represents a (local) time of day, independent of any particular day"""
+
     type_class_uri = XSD.dateTime
     type_class_curie = "xsd:dateTime"
     type_name = "time"
     type_model_uri = SSSOM.CS.Time
 
 
 class Date(XSDDate):
-    """ a date (year, month and day) in an idealized calendar """
+    """a date (year, month and day) in an idealized calendar"""
+
     type_class_uri = XSD.date
     type_class_curie = "xsd:date"
     type_name = "date"
     type_model_uri = SSSOM.CS.Date
 
 
 class Datetime(XSDDateTime):
-    """ The combination of a date and time """
```

Comment:
@matentzn how are these generated? It would be good to fix this upstream so it's already flake8 compliant

Reply:
There is a related issue: https://github.com/mapping-commons/sssom-py/issues/121